R/SingleR.visualizations.R

```diff
@@ -0,0 +1,155 @@
+#' Plot expression of a cell versus a reference cell/sample
+#'
+#' @param sc.data Numeric matrix of single-cell expression values (usually log-transformed
+#' or otherwise variance-stabilized), where rows are genes and columns are cells.
+#' Alternatively, a \linkS4class{SingleCellExperiment} object containing such a matrix.
+#' @param sc.id Integer scalar specifying the index of the target cell to use.
+#' Alternatively, the name of the cell.
+#' @param train.data Numeric matrix of reference dataset expression values (usually log-transformed
+#' or otherwise variance-stabilized), where rows are genes and columns are cells.
+#' Alternatively, a \linkS4class{SingleCellExperiment} object containing such a matrix.
+#' @param train.id Integer scalar specifying the reference cell/sample to use.
+#' @param assay.type.sc Integer scalar or Character specifying the assay of \code{sc.data} containing the relevant assay if provided as a \linkS4class{SingleCellExperiment}.
+#' @param assay.type.train Integer scalar or Character specifying the assay of \code{train.data} containing the relevant assay if provided as a \linkS4class{SingleCellExperiment}.
+#' if \code{sc.data} or \code{train.data} is a \linkS4class{SingleCellExperiment} object.
+#' @return a gpplot scatterplot of the expression of indidivual genes, with linear regression and Spearman pariwaise correlation coefficient
```

Comment:
A few typos here.

Reply:
oops.  That 3rd line was meant to be removed.  (I'd previously only had 'assay.type' input, but decided to allow separate control.)
Fixed with a bit better wording.